leshan-server-core/src/main/java/org/eclipse/leshan/server/observation/ObservationListener.java

```diff
@@ -30,5 +30,13 @@
      * @param observation the observation for which new data are received
      * @param reponse the lwm2m response received
      */
-    void newValue(Observation observation, ObserveResponse response);
+    void onResponse(Observation observation, ObserveResponse response);
```

Comment:
We need to make sure that we document this properly in these methods' JavaDoc because I am afraid that `onError()` might otherwise be easily mistaken as a handler for 4.xx responses...